src/display/editor/tools.js

```diff
@@ -285,35 +285,46 @@ class KeyboardManager {
  */
 class ClipboardManager {
   constructor() {
-    this.element = null;
+    this.elements = null;
```

Comment:
Slightly unrelated to this patch, but should this property perhaps be *private* instead?